js/forum/src/main.js

```diff
@@ -9,8 +9,10 @@ import EmojiAreaButton from 'clarkwinkelmann/emojionearea/components/EmojiAreaBu
 
 app.initializers.add('clarkwinkelmann-emojionearea', () => {
     extend(TextEditor.prototype, 'controlItems', function (items) {
-        var emojiButton = new EmojiAreaButton;
-        emojiButton.textEditor = this;
+        const emojiButton = new EmojiAreaButton({
```

Comment:
I guess when it is an item list, you can pass the class by itself. But if you were to do a `m('div', ExampleComp.component());`